design/src/client/configuration.md

```diff
@@ -0,0 +1,237 @@
+Client configuration for the orchestrator
+-----------------------------------------
+
+This document discusses two areas of improvement for configuring an SDK client:
+- Support for operation-level configuration
+- Support for runtime components required by the orchestrator
+
+We have encountered use cases where configuring a client for a single operation invocation is necessary ([example](https://github.com/awslabs/aws-sdk-rust/issues/696)). At the time of writing, this feature is not yet supported, but operation-level configuration will address that limitation.
+
+As described in [RFC 34](https://github.com/awslabs/smithy-rs/blob/main/design/src/rfcs/rfc0034_smithy_orchestrator.md), the orchestrator uses configured components for an SDK to handle messages between the client and a service. They include:
+- `RetryStrategy`: Configures how requests are retried.
+- `TraceProbes`: Configures locations to which SDK metrics are published.
```

Comment:
No, we do not yet.  While the details of `TraceProbes` need to be worked out, since we already have `tracing` in Rust, the priority of `TraceProbes` will probably be lower.